hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/OzoneTestUtils.java

```diff
@@ -161,4 +161,37 @@ public static void closeContainer(StorageContainerManager scm,
             container.getState() == HddsProtos.LifeCycleState.CLOSED,
         200, 30000);
   }
+
+  /**
+   * Flush deleted block log & wait till something was flushed.
+   */
+  public static void waitForDeletedBlockLog(StorageContainerManager scm)
```

Comment:
suggest to rename the method as "flushAndWaitForDeletedBlockLog()"

Reply:
done